srcTest/testFunctions.test.js

```diff
@@ -0,0 +1,10 @@
+import { search_pattern } from "./../src/Class Specs/Class_sport";
+import { test, expect } from "vitest";
+
+test("Check data to be fasle when comparing Test with TEST", () => {
+  expect(search_pattern("Test", "TEST")).toBe(false);
+});
```

Comment:
Other than adding only one test, maybe add the two similar test that have the same function and test them with different style of words, AbAb, ASDasd, Asdsd, and more.

Reply:
You are correct, however logically their should not be any errors, maybe except when passing in wrong arguments.